src/utils/static-server.mjs

```diff
@@ -21,7 +21,8 @@ export const startStaticServer = async ({ settings }) => {
   })
 
   server.addHook('onRequest', (req, reply, done) => {
-    reply.header('X-Powered-by', 'netlify-dev')
```

Comment:
should we add the `Server: Netlify` header? or `Server: Netlify Dev`, as we call it in https://docs.netlify.com/cli/get-started/#get-started-with-netlify-dev?

Reply:
Unless there's a reason to differentiate them, I'd rather use the same value in both environments.

Reply:
okay, but right now we're not setting `Server` anywhere, right? should we?

Reply:
We're not. Production gives you `server: Netlify` and CLI gives you `x-powered-by: netlify-dev`.

Reply:
that's what I figured. This PR is about moving the CLI closer to prod together, so let's add `reply.header("Server", "Netlify")`

Reply:
I'm confused. That's what we're doing [here](https://github.com/netlify/cli/pull/5490/files#diff-5c5309633069037040a13b82d7abcfffe577c7516e6d356e46f407d729aa98a8R392), except we're doing it for all responses and not just the ones served by the static server.

Reply:
I was confused. Didn't see that, sorry.